src/Diagnostics/FxCop/System.Runtime.Analyzers/Core/System.Runtime.Analyzers.Common.props

```diff
@@ -12,6 +12,7 @@
       -Microsoft.Design#CA1014;
       -Microsoft.Design#CA1016;
       -Microsoft.Design#CA1017;
+      -Microsoft.Design#CA1018;
```

Comment:
It should also be CA1813. I'll move CA1813 in my next PR (which is moving one more rule).